src/helpers/joinLikesAccomodation.js

```diff
@@ -0,0 +1,10 @@
+const newAccommodation = (data, likes) => {
+  const accommodation = data.map((item, index) => {
+    const findLikesDislikes = likes[index];
+    item.LikesDislikes = findLikesDislikes;
+    return item;
+  });
+  return accommodation;
+};
+
```

Comment:
@NiyoEric Would you please give me a hint of what this function does?

Reply:
I did it because paginating forward was not displaying the total average rate for each accommodation on a page. Pagination displays 5 items. I was displaying all data according to accommodation data length If the item’s index is greater than 5 its data was not displayed because the length of data saved in pagination will always be less than five. I have created a helper file to solve the problem.

Reply:
It makes sense, Thanks @NiyoEric.